bridge_common.spec

```diff
@@ -0,0 +1,67 @@
+%define pkg_name tendrl-bridge-common
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Common Module for All Bridges
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: LGPL2.1
```

Comment:
Please use a License value from the table at https://fedoraproject.org/wiki/Licensing:Main#Good_Licenses ("Short Name" column). For example, `License: LGPLv2`